docs/resources/aws_ecs_service.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About the aws_ecs_service Resource
+platform: aws
+---
+
+# aws\_ecs\_service
+
+Use the `aws_ecs_service` InSpec audit resource to test properties of a single specific Athena Work Group.
+
+The AWS::ECS::Service resource creates an Amazon Elastic Container Service (Amazon ECS) service that runs and maintains the requested number of tasks and associated load balancers.
+
+## Syntax
+
+Ensure that a services exists.
+    describe aws_ecs_service(cluster: "default", services: ["arn:aws:ecs:us-east-1:012345678910:service/ecs-simple-service"]) do
+      it { should exist }
+    end
+
+## Parameters
+
+`cluster, services` _(required)_
+
+For additional information, see the [AWS documentation on ECS Service](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
```

Comment:
It seems like `cluster`, `services` and `failures` are the parameters and the properties are `cluster_arn`, `created_at`, `deployment_configuration`, etc...

Reply:
Yes. That why i have added a line "For additional information, see the [AWS documentation on ECS describe services method](https://docs.aws.amazon.com/sdk-for-ruby/v2/api/Aws/ECS/Client.html#describe_services-instance_method).". So that one can go to this docs and see the following.

Reply:
Those properties need to be added to the text so users know what they can test.